cnxarchive/tests/test_database.py

```diff
@@ -780,8 +780,12 @@ def test_republish_collection(self, cursor):
         collection_ident = cursor.fetchone()[0]
         cursor.connection.commit()
 
-        new_ident = republish_collection(
-            3, collection_ident, testing.fake_plpy)
+        replublished_submittor = "replublished_submittor"
+        replublished_submitlog = "replublished_submitlog"
+
+        new_ident = republish_collection(replublished_submittor,
```

Comment:
@karenc Thanks for pointing it out. I see that I cam not checking against the row with the module ident. I will fix it later.